lib/unparser/emitter/literal/primitive.rb

```diff
@@ -7,10 +7,41 @@ class Primitive < self
 
         children :value
 
+        # Emitter for string literals
+        class String < self
+          handle :str
+
+        private
+
+          # Dispatch value
+          #
+          # @return [undefined]
+          #
+          # @api private
+          #
+          def dispatch
+            if value.include? "'"
```

Comment:
I also think there is a constant for `'` already we should re-use to avoid object clutter.